docs/source/quicktour.mdx

```diff
@@ -15,10 +28,8 @@ To evaluate `GPT-2` on the Truthful QA benchmark, run:
 
 ```bash
 lighteval accelerate \
-     --model_args "pretrained=gpt2" \
-     --tasks "leaderboard|truthfulqa:mc|0|0" \
-     --override_batch_size 1 \
-     --output_dir="./evals/"
+     "pretrained=gpt2" \
+     "leaderboard|truthfulqa:mc|0|0"
```

Comment:
you're removing the output dir from all commands, why?
I think it's good for users to be able to interact with outputs directly

Reply:
Because it was a required arg and is not anymore. I feel like having users type the less possible while having the option for more advanced users is better but I can change it